gensim/models/keyedvectors.py

```diff
@@ -814,17 +948,18 @@ def nbow(document):
         # Compute nBOW representation of documents.
         d1 = nbow(document1)
         d2 = nbow(document2)
-
```

Comment:
Please keep the blank lines that separate code blocks (easier to read).

Reply:
Fixed